express-api/src/routes/reportsRouter.ts

```diff
@@ -0,0 +1,28 @@
+import controllers from '@/controllers';
+import express from 'express';
+
+const router = express.Router();
+
+router.route('/projects').get(controllers.getSpreadsheetProjectsReports);
+
+/**
+ * I'm pretty sure this is not necessary but perhaps someone can weigh in. Why do we need an endpoint specifically for
+ * surplus properties reports when surplus is simply a classification of properties? I feel like you should really only
+ * need to apply a filter on the general properties endpoint. The endpoints do actually function differently but they appear
+ * to produce a similar end result so not sure what the distinction is for.
+ */
+//router.route('/projects/surplus/properties').get(controllers.getSpreadsheetSurplusPropertiesReport);
```

Comment:
Agreed. As far as I know, it would just be properties that have a classification matching one of the surplus classifications.